apps/molgenis-components/src/components/forms/baseInputs/BaseInputDecimal.vue

```diff
@@ -27,16 +27,16 @@ export default {
   methods: {
     emitIfValid(event) {
       const value = parseFloat(event.target.value);
-      if (event.target.value === "") {
-        this.$emit("input", null);
+      if (event.target == null || event.target.value == null) {
```

Comment:
===
might also consider event.target?.value === null
Since event is not an object we made 'event.target' is probably undefined instead of null if it doesn't exist. So I think this should be:
if (event.target?.value)